### DISCUSSION TITLE
yourls 部署好了之后生成一个短链访问问题？

### DISCUSSION THREAD

**Discussion by xuhongzhang123** [YOURLS/YOURLS#2991] [Q&A] [answered]
A short chain is generated to access. The result is that the HTTP response is 200 instead of 301, and the original URL is nested within the short URL that is, the iframe is nested within the target URL.

eg http://tokunion.com/qtjg5

Let's have a look at it. Click the link above to see what happens

**Comment by matthewpkemp** [comment, marked as answer]
I'm not sure what the question is.  I see the original 301 from tokunion.com which is acting as expected.
![Screen Shot 2021-07-14 at 5 09 54 PM](https://user-images.githubusercontent.com/852761/125701512-e29b6a29-171e-407b-b3ea-aba929fb30a8.png)
